aspnetcore/blazor/performance/browser-developer-tools-profiling.md

```diff
@@ -1,22 +1,22 @@
 ---
-title: ASP.NET Core Blazor WebAssembly developer tools performance profiling and diagnostic counters
+title: ASP.NET Core Blazor WebAssembly browser developer tools diagnostic profiling
```

Comment:
please drop `profiling`. I would not call GC dump or counters "profiling". But all of it is diagnostics in my mind.